parcels/field.py

```diff
@@ -1,7 +1,7 @@
 from parcels.tools.loggers import logger
 from parcels.tools.converters import unitconverters_map, UnitConverter, Geographic, GeographicPolar
 from parcels.tools.converters import TimeConverter
-from parcels.tools.error import FieldSamplingError, TimeExtrapolationError
+from parcels.tools.error import FieldSamplingError, FieldOutOfBoundError, TimeExtrapolationError
```

Comment:
Do we need to implement the same separation between SamplingError and OutOfBoundError in `index_search.h` too?

Reply:
See 7ecfc02